content/blog/2016/2016-12-13-jenkins-puppet-enterprise-plugin.adoc

```diff
@@ -0,0 +1,271 @@
+---
+layout: post
+title: "Continuous delivery with Jenkins and Puppet Enterprise"
+tags:
+- jenkins
+- continuousdelivery
+- puppet
+author: ccaum
+---
+
+During PuppetConf 2016, myself and Brian Dawson from CloudBees announced the
+link:https://wiki.jenkins-ci.org/display/JENKINS/Puppet+Enterprise+Pipeline+Plugin[Puppet
+Enterprise plugin for Jenkins Pipeline.] Let's take a look at how the plugin
+makes it trivial to use Puppet to perform some or all of the deployment tasks
+in continuous delivery pipelines.
+
+Jenkins Pipeline introduced an amazing world where pipelines are defined and
+managed from the version control repository that houses the code the pipeline
+delivers. This is a powerful idea, and one I felt complemented Puppet's
+automation strengths. I wanted to make it trivial to control Puppet
+Enterprise's orchestration and infrastructure code management capabilities, as
+well as control Hiera data and use PuppetDB as a source of truth – all from a
```

Comment:
Probably worth it to make Hiera and PuppetDB into links to where they're described for people who don't know wtf is going on.